src/frequenz/sdk/_internal/_math.py

```diff
@@ -0,0 +1,13 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Internal math tools."""
+
+import functools
+import math
+
+# A value of 1e-9 is a commonly used absolute tolerance to balance precision
+# and robustness for floating-point numbers comparisons close to zero. Note
+# that this is also the default value for the relative tolerance.
+# For more technical details, see https://peps.python.org/pep-0485/#behavior-near-zero
+_is_close_to_zero = functools.partial(math.isclose, b=0.0, abs_tol=1e-9)
```

Comment:
If this is now in a separate module I would make it regular function that calls `math.isclose()`, so it can be properly documented (probably including what's in the comments as part of the documentation). Also, remove the leading `_`, as this is a **public** function (in a **private** module).

Reply:
True, the functools was helpful to use it in the previous case but in this independent private module makes sense to have a regular function. I have already updated it accordingly